vpp/algorithms/Line_tracker_4_sfm/Fast_DHT/fast_hough.hh

```diff
@@ -0,0 +1,54 @@
+#ifndef HOUGH_IMAGE_HH
+#define HOUGH_IMAGE_HH
+
+
+
+#include <ctime>
+#include <cstdlib>
+
+#include <opencv2/highgui.hpp>
+#include <vpp/vpp.hh>
+#include <Eigen/Core>
+#include <vpp/utils/opencv_bridge.hh>
+#include <vpp/utils/opencv_utils.hh>
+#include "boost/intrusive/list.hpp"
+#include "dense_one_to_one_hough.hh"
+#include "vpp/algorithms/Line_tracker_4_sfm/symbols.hh"
+#include "vpp/algorithms/Line_tracker_4_sfm/miscellanous/define.hh"
+
+
+
+using namespace vpp;
+using namespace std;
+using namespace Eigen;
+using namespace iod;
+using namespace cv;
+
+namespace vpp{
+
```

Comment:
It's not clear which function are part of the API you're providing and which one are only internal.